Update free-courses-hi.md (#10540)
EbookFoundation/free-programming-books: courses/free-courses-hi.md

### iOS

> *[iOS App Devlopment for Beginner in Hindi](https://www.youtube.com/playlist?list=PL5PR3UyfTWvcRhz4ms-nNoGja2-DkA_6T) - iOS Academy

Reviewer: ```suggestion
* [Use the title from the Playlist](https://www.youtube.com/playlist?list=PL5PR3UyfTWvcRhz4ms-nNoGja2-DkA_6T) - iOS Academy
```
Don't invent titles! Use the title given by the resource.

Author: fixed
